src/components/stimuli/ClickAccuracyTest.tsx

```diff
@@ -14,15 +16,45 @@ const chartSettings = {
   width: 850,
 };
 
+interface ClickAccuracyTest {
+  distance: number;
+  speed: number;
+  clickX: number;
+  clickY: number;
+}
+
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const ClickAccuracyTest = ({ parameters, trialId }: { parameters: any, trialId: string }) => {
+const ClickAccuracyTest = ({ parameters, trialId, setAnswer }: { parameters: any, trialId: string, setAnswer: ({trialId, status, provenanceGraph, answers} : {trialId: string, status: boolean, provenanceGraph?: ProvenanceGraph<any, any, any>, answers: Record<string, any>}) => void }) => {
```

Comment:
Can we make this type look nicer? Maybe extract the type to a locally declared type above line 27?

Reply:
This could also be part of the API that we pass down to these stimulus controllers to help our users integrate with the platform

Reply:
So I moved the types of the entire parameters for these stimulus components into the types, typed the clickAccuracy as StimulusParameters, and did some type magic in the flags to make that work. So now we actually have some intelisense in the Stimulus components, which is cool!